package.json

```diff
@@ -30,11 +31,11 @@
     "@s-ui/mono": "1",
     "@s-ui/precommit": "2",
     "@s-ui/studio": "8",
-    "fs-extra": "8.1.0",
-    "globby": "10.0.1",
-    "husky": "0.13.4",
-    "validate-commit-msg": "2.12.2",
-    "walker": "1.0.7"
+    "fs-extra": "8",
```

Comment:
Why we have updated the version pointing the major one? Normally we do that only on our private packages or SUI packages.

Reply:
Fixed!